Countries Info/ViewModels/CountriesListViewModel.swift

```diff
@@ -6,31 +6,40 @@
 //
 
 import Foundation
+import Combine
 
 protocol CountriesListViewModelProtocol {
-    var countries: [Country]? { get }
+    var countriesPublisher: Published<[Country]?>.Publisher { get }
     
-    func fetchResult(completion: @escaping() -> Void)
     func numberOfItem() -> Int
     func cellViewModel(indexPath: IndexPath) -> CountryCollectionViewCellViewModelProtocol?
+    func fetch()
 }
 
-class CountriesListViewModel: CountriesListViewModelProtocol {
+class CountriesListViewModel: CountriesListViewModelProtocol, ObservableObject {
     
-    var countries: [Country]?
+    @Published var countries: [Country]?
```

Comment:
Do we need this array to be optional? 
Please check if we can use empty array as a default value, thanks